libs/tools/generator/components/src/username-generator.component.html

```diff
@@ -59,7 +59,7 @@ <h2 bitTypography="h6">{{ "options" | i18n }}</h2>
         </bit-form-field>
       </form>
       <tools-catchall-settings
-        *ngIf="(algorithm$ | async)?.id === 'catchall'"
+        *ngIf="(showAlgorithm$ | async)?.id === 'catchall'"
```

Comment:
```suggestion
        *ngIf="(showAlgorithm$ | async)?.id === Algorithm.catchall"
```
⛏️  This and the below ones could also be updated as in the `credential-generator.component` to use the `Algorithm` instead of a `string` to compare.